src/ui/components/filters/filterboxcomponent.js

```diff
@@ -247,6 +257,16 @@ export default class FilterBoxComponent extends Component {
     }
   }
 
+  /**
+   * Handle filter node removal in a child FilterOptions component.
+   * @param {number} index
+   * @param {FilterNode} filterNode
+   */
+  onFilterNodeRemoval (index, filterNode) {
+    this._filterNodes[index] = filterNode;
+    this._saveFilterNodesToStorage();
```

Comment:
I'm a bit confused, how does this method work? I would've thought this method first created a version of _filterNodes without this particular filterNode, called the remove method for that filterNode, then called saveFilterNodesToStorage. If I remove a filterNode here, how is the corresponding child filter getting re-rendered?

Reply:
_saveFilterNodesToStorage does that, I guess the name isn't terribly accurate

Reply:
This is also only used for facets, so I should probably rename it

Reply:
ok this method name is garbage deleting it like we talked about

Reply:
updated!